website/content/en/_index.md

```diff
@@ -3,28 +3,15 @@ title: ClusterLink
 ---
 
 {{< blocks/cover image_anchor="smart" height="auto" color="secondary" >}}
- {{% clusterlink/intro logo="clusterlink_logo.png" buttontext="Get Started" buttonlink="/docs/getting-started" %}}Securely connecting<br />applications across multi-cloud{{% /clusterlink/intro %}}
+ {{% clusterlink/intro logo="ClusterLink_Logo.png" buttontext="Get Started" buttonlink="docs/getting-started" %}}Securely connecting<br />applications across multi-cloud{{% /clusterlink/intro %}}
```

Comment:
nit: I don't think we should use mixed case file names but rather keep all lower case for consistency.
suggest renaming `ClusterLink_Logo.png` back to the original name (and overwriting instead of adding/removing the previous) and not changing this `_index.md` file

Reply:
Will change back the name